Integrations/integration-VxStream.yml

```diff
@@ -284,6 +288,33 @@ script:
         return res;
     }
 
+    function detonateFile(entryId, environmentId, delay, timeout) {
+
+        var hash = submitFile(entryId, environmentId).substring(10);
+
+        delayTime = parseInt(delay);
+        timeOut = parseInt(timeout);
+        var waitTime = delayTime;
+
+        sleep(delayTime);
+
+        while (waitTime<timeOut) {
+            res = scan(hash);
+            if (res.response[0]) {
+                return scanToEntry(res);
+            } else {
+                waitTime = waitTime + delayTime;
+                sleep(delayTime)
```

Comment:
we have a wait function in jshttp that doesn't do busy waiting, use it instead of your implementation